homeassistant/core.py

```diff
@@ -1960,10 +1961,10 @@ async def async_update(self, **kwargs: Any) -> None:
         await self.async_store()
         self.hass.bus.async_fire(EVENT_CORE_CONFIG_UPDATE, kwargs)
 
-    async def async_load(self) -> None:
-        """Load [homeassistant] core config."""
+    @callback
+    def _get_store(self) -> Store[dict[str, Any]]:
```

Comment:
Having it available from a single location will avoid discrepencies when adding minor version and migration.